### PR TITLE
Fix: Fixed Register signup process and login signin process

### LINKED ISSUE #10: Bug : Username asking for email
Username has input field of email

### PR DESCRIPTION
**Pull request by yube01**
Register signup process fixed 
Register page redirect to login page after successful login process
In login page , if login process is success user is redirected to home page

fixes #10

### CODE REVIEW

**Review by ojasaklechayt — COMMENTED** [review]
Changes reviewed

File: frontend/app/login/page.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ export default function Login() {
                 credentials: 'include',
             });
             console.log('Login successful:', response);
-            if (response.statusText === 'OK') {
+            if (response.status === 200) {
```
[the remark this answers is not in the document]
Comment (yube01) [reply frontend/app/login/page.js:36]: StatusText was empty so I changed to status

[the remark this answers is not in the document]
Comment (ojasaklechayt) [reply frontend/app/login/page.js:36]: That is required for authentication. It is giving " OK " when use is logged in. So it is required field.

**Comment by yube01** [comment]
login page code changed back to previous one

**Comment by ojasaklechayt** [comment]
@yube01 it's same

**Review by yube01 — COMMENTED** [review]
Changed back to previous state

File: frontend/app/login/page.js [hunk 2]
```diff
@@ -76,7 +76,7 @@ export default function Login() {
                                                 onChange={handleInputChange} />
                                         </div>
 
-                                        <button type="submit" className="w-[50%] bg-white text-black font-bold rounded-md py-2 mt-7 px-3 sm:px-4 duration-50 text-sm sm:text-base font-poppins hover:bg-gray-300 hover:cursor-pointer transition duration-300 ease-in-out">Log In</button>
+                                        <button type="submit" onClick={handleLogin} className="w-[50%] bg-white text-black font-bold rounded-md py-2 mt-7 px-3 sm:px-4 duration-50 text-sm sm:text-base font-poppins hover:bg-gray-300 hover:cursor-pointer transition duration-300 ease-in-out">Log In</button>
```
Comment (ojasaklechayt) [inline frontend/app/login/page.js:79]: remove `onClick = {handleLogin}` also, as we already have `type = " submit " ` which will automatically handle the changes. We already have this in the form tag `onSubmit={handleLogin}`

**Review by yube01 — COMMENTED** [review]
onclick event removed

**Pull request merged**